registrations/models.py

```diff
@@ -697,33 +697,40 @@ def get_registration_message(self, subject, cleaned_body, plain_text_body):
             [self.email],
         )
 
-    def send_notification(self, notification_type):
+    def get_notification_message(self, notification_type):
         [_, linked_registrations_ui_locale] = get_ui_locales(self.service_language)
 
+        if notification_type == SignUpNotificationType.EVENT_CANCELLATION:
+            email_template = "event_cancellation_confirmation.html"
+        elif notification_type == SignUpNotificationType.CANCELLATION:
+            email_template = "cancellation_confirmation.html"
+        else:
```

Comment:
IMO in cases like this, it's better to be defensive and check explicitly all valid cases, and then raise an exception in the `else` block for possible invalid types, so calling the function incorrectly won't be silently ignored.